### PR TITLE
document improvements

### PR DESCRIPTION
**Pull request by voroyam**
- changed name of the document
- made the improvements suggested by Interns
- removed unnecessary warnings and notes
- changed the LDAPS section back to the original state

### CODE REVIEW

File: modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 1]
```diff
@@ -18,19 +18,11 @@
 :selinux-url: https://selinuxproject.org/page/Main_Page
 :service-ldap-with-tls-url: https://ubuntu.com/server/docs/service-ldap-with-tls
 :ldap-over-ssl-ldaps-certificate-url: https://social.technet.microsoft.com/wiki/contents/articles/2980.ldap-over-ssl-ldaps-certificate.aspx
+:marketplace-URL: https://marketplace.owncloud.com/apps/user_ldap
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc:21]:
```suggestion
```

File: modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 2]
```diff
@@ -866,3 +838,7 @@ TIP: Do not do this in production.
 When ownCloud is not able to contact the main LDAP server, ownCloud assumes it is offline and will not try to connect again for the time specified in "Cache Time-To-Live". 
 If you have a backup server configured ownCloud will connect to it instead. 
 When you have scheduled downtime, check btn:[Disable Main Server] to avoid unnecessary connection attempts.
+
+
+
+
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc:844]:
```suggestion
```
I guess that the blank lines at the end are not intentional 😄

**Comment by mmattel** [comment]
Mind to explain why taking back the LDAPS part? What is not working?

**Comment by voroyam** [comment]
> Mind to explain why taking back the LDAPS part? What is not working or what specifically are you adressing ?

because mine was: clear 3 steps and it was tested. I was not included on your change. It was simply merged.

**Review by mschreml — APPROVED** [review]

**Pull request merged**